.github/workflows/publish_docker.yaml

```diff
@@ -7,31 +7,15 @@ on:
 env:
   IMAGE_NAME: jupyter
 jobs:
-  tag-new-version:
-    runs-on: ubuntu-latest
-    outputs:
-      tag: ${{ steps.tag.outputs.new_version }}
-    steps:
-      - uses: actions/checkout@main
-        with:
-          fetch-depth: 0
-      - name: Bump version and push tag
-        id: tag
-        uses: mathieudutour/github-tag-action@v4.5
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          default_bump: false
-
-  build-and-publish-docker-image:
-    runs-on: ubuntu-latest
-    name: Build and publish docker image
-    needs: tag-new-version
-    if: needs.tag-new-version.outputs.tag
+  build:
+    runs-on: ubuntu-20.04
     steps:
     - name: Checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@master
```

Comment:
This seems weird. I think it's OK not to pin to a sha for official actions like this, but don't we want to specify a version tag at least?

Reply:
huh, I didn't do that one on purpose, will pin to @v2